src/main/webapp/app/shared/components/atoms/button/button.component.scss

```
@@ -0,0 +1,157 @@
+@import '../../../../../content/scss/tokens';
+
+$primary-default: $primary-300; //var(--p-primary-color, #{$primary-300});//
+$secondary-default: $secondary-300; // var(--p-primary-color);
+$info-default: $info-500; //var(--p-info-color);
+$positive-default: $positive-500; //var(--p-success-color);
+$warning-default: $warning-500; //var(--p-warn-color);
+$negative-default: $negative-500; //var(--p-danger-color);
+$background-disabled: $neutral-200;
+
+.p-button {
+  overflow: unset;
+  border: 0.06rem solid var(--p-primary-color) !important;
+}
+
+@mixin state-styles($base-color, $text-color: black) {
+  &:hover {
+    background: darken($base-color, 10%) !important;
+  }
+
+  &.disabled,
+  &:disabled {
+    background: lighten($base-color, 40%) !important;
+    border-color: var(--background-disabled) !important;
+    color: $text-color !important;
+    pointer-events: none;
+    opacity: 0.6;
+  }
+}
+
+@mixin outlined-button($color) {
+  background: white !important;
+  border: 0.06rem solid $color !important;
+  color: $color !important;
+
+  &:hover {
+    background: $color !important;
+    color: white !important;
+  }
+
+  &.disabled,
+  &:disabled {
+    background: white !important;
+    border-color: var(--background-disabled) !important;
+    color: var(--text-on-disabled) !important;
+    pointer-events: none;
+    opacity: 0.6;
+  }
+}
+
+@mixin filled-button($color, $text-color: black, $text-color-disabled: black) {
```

Why do we need "$text-color-disabled"? I would suggest removing it if it is not being used anywhere.
```suggestion
@mixin filled-button($color, $text-color: black) {
```

---

This is all solved through the primeng theme now